examples/fastify/README.md

```diff
@@ -27,6 +27,10 @@ npm install
    $ npm run docker:start
    ```
 
+Note: If you are happen to run the example on a Apple Silicon system, you might
+get an error similar to `no matching manifest for Linux/arm64/v8 in the manifest list entries`
+in that case you can resolve this error by adding `platform: linux/amd64` to each service in `docker/docker-compose.yaml`.
```

Comment:
Did you by any chance look into why this happens? is it possible that this is relevant to other examples as well?

Maybe upgrading the docker image tags to something more up-to-date can resolve the issue? 
opentelemetry-collector:0.38.0 is

Reply:
Yeah, I will update it and see if the issue still occurs. The problem is that the docker image is not a multi-arch image that includes arms.

Reply:
Removed it as with a newer version of otel-collector it worked without issue